runtime/parser2/expression.go

```diff
@@ -295,7 +295,7 @@ func init() {
 
 	defineExpr(literalExpr{
 		tokenType: lexer.TokenIdentifier,
-		nullDenotation: func(_ *parser, token lexer.Token) ast.Expression {
+		nullDenotation: func(p *parser, token lexer.Token) ast.Expression {
 			switch token.Value {
 			case keywordTrue:
```

Comment:
I wonder if it would be better to emit a `TokenKeywordTrue` etc. at the lexer level. Also wonder if there is merit to having them as their own tokens versus under the `Identifier` token. Just thinking out loud!